desktop-gnome/gtk4-layer-shell/autobuild/defines

```diff
@@ -0,0 +1,14 @@
+PKGNAME="gtk4-layer-shell"
+PKGSEC=x11
+PKGDES="Library to create panels and other desktop components for Wayland"
```

Comment:
Maybe remove "other"? cc @MingcongBai

Reply:
Library to create Wayland desktop panels and components